bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -474,7 +516,7 @@ impl OlmMachine {
         device_changes: DeviceLists,
         key_counts: HashMap<String, i32>,
         unused_fallback_keys: Option<Vec<String>>,
-    ) -> Result<String, CryptoStoreError> {
+    ) -> Result<SyncChangesResult, CryptoStoreError> {
```

Comment:
We should also update the documentation to reflect the new return type.

Reply:
The return type itself contains the documentation. Though I did fix it a bit in https://github.com/matrix-org/matrix-rust-sdk/pull/2297/commits/e603c96b72fd011f9cd42a4499789465bc37f1b6.